api-interop-layer/data/satellite.test.js

```diff
@@ -5,44 +5,46 @@ import getSatelliteData from "./satellite.js";
 describe("satellite metadata module", () => {
   const sandbox = createSandbox();
   const response = {
+    status: 200,
     json: sandbox.stub(),
   };
 
   beforeEach(() => {
+    response.status = 200;
     fetch.resolves(response);
   });
 
   describe("returns a URL if everything goes well", () => {
     it("for GOES-16 imagery", async () => {
       response.json.resolves({ meta: { satellite: "GOES-West" } });
-      const actual = await getSatelliteData({ grid: { wfo: "wfo" } });
+      const actual = await getSatelliteData({ grid: { wfo: "wfo1" } });
```

Comment:
any reason why we changed to `wfo1` `wfo2` etc?

Reply:
Yes! When debugging the tests, I set the logging to verbose so I could get a better sense of what was going on, and it was helpful to be able to differentiate which test was causing which set of log messages.